robosuite/wrappers/gym_wrapper.py

```diff
@@ -83,15 +84,20 @@ def _flatten_obs(self, obs_dict, verbose=False):
                 ob_lst.append(np.array(obs_dict[key]).flatten())
         return np.concatenate(ob_lst)
 
-    def reset(self):
+    def reset(self, seed=None, options=None):
         """
-        Extends env reset method to return flattened observation instead of normal OrderedDict.
+        Extends env reset method to return flattened observation instead of normal OrderedDict and optionally resets seed
 
         Returns:
             np.array: Flattened environment observation space after reset occurs
         """
+        if seed is not None:
+            try:
```

Comment:
the try and except might be overkill - why not a simple `isinstance(seed, int)` check?

Reply:
> the try and except might be overkill - why not a simple `isinstance(seed, int)` check?

That's fair! I can simplify it to that.

> also, should seed setting happen in the reset function? I'm not super familiar with the new gymnasium api, but I would think that the seed should be set before constructing the environment, or maybe in the environment init

In the new gymnasium api, they have seed passed in to the `env.reset`. From the docs: "Therefore, [reset()](https://gymnasium.farama.org/api/env/#gymnasium.Env.reset) should (in the typical use case) be called with a seed right after initialization and then never again."